### PR TITLE
chore: move hardhat-solidity-tests to the EDR repo

### PR DESCRIPTION
**Pull request by fvictorio**

### CODE REVIEW

**Review by agostbiro — CHANGES_REQUESTED** [review]
Thanks looks good overall, but could you also include the usage instructions (and update if necessary) from the [original repo?](https://github.com/NomicFoundation/hardhat-solidity-tests?tab=readme-ov-file#how-to-use)

**Comment by fvictorio** [comment]
Good catch, thanks!

While writing the readme I also realized that it makes sense to have the example in this repo too, so no `pnpm link` is needed at all for running it.

File: hardhat-solidity-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Temporary directory to experiment with a Hardhat plugin that uses EDR's solidity tests feature.
+
+This directory can be removed once the plugin is handed over to the Hardhat team.
+
+## How to use
+
+1. Clone this repo and [the example](https://github.com/NomicFoundation/hardhat-solidity-tests-example)
```
Comment (agostbiro) [inline hardhat-solidity-tests/README.md:7]:
```suggestion
1. Clone this repo and [the usage example](https://github.com/NomicFoundation/hardhat-solidity-tests-example) to `./example`
```

Comment (agostbiro) [reply hardhat-solidity-tests/README.md:7]: Note that the example repo is private, we should make that public probably

**Review by agostbiro — APPROVED** [review]
Thanks LGTM with one suggestion for improvement

File: hardhat-solidity-tests/README.md (hunk 1 above)
Comment (fvictorio) [reply hardhat-solidity-tests/README.md]: No, that was my bad. There's no need to clone that repo anymore. I simplified the steps now.

Comment (agostbiro) [reply hardhat-solidity-tests/README.md]: Ah ok cheers! Good to merge from my side

**Pull request merged**